terraform/backend_xi.tf

```diff
@@ -0,0 +1,43 @@
+module "backend_xi" {
+  source = "git@github.com:trade-tariff/trade-tariff-platform-terraform-modules.git//aws/ecs-service?ref=aws/ecs-service-v1.2.0"
+
+  service_name  = var.service_name
+  service_count = var.service_count
+  environment   = var.environment
+  region        = var.region
+
+  cluster_name     = "trade-tariff-cluster-${var.environment}"
+  subnet_ids       = data.aws_subnets.private.ids
+  security_groups  = [data.aws_security_group.this.arn]
+  target_group_arn = data.aws_lb_target_group.this.arn
+
+  min_capacity = var.min_capacity
+  max_capacity = var.max_capacity
+  docker_image = data.aws_ssm_parameter.ecr_url.value
+  docker_tag   = var.docker_tag
+  skip_destroy = true
+
+  # backend_xi_vars
+  service_environment_config = [
```

Comment:
Both services will need a base set of environment variables, plus the ones that are specific to UK and XI on top. It might make more sense to create a local that has the basic config shared between both, then add this to the list for each of the two services.